lib/internal/watch_mode/files_watcher.js

```diff
@@ -85,13 +85,20 @@ class FilesWatcher extends EventEmitter {
     return [...this.#watchers.keys()];
   }
 
+  [Symbol.for('emitChangeEventWithoutFilename')]() {
+    this.#watchers.values().forEach(watcher => watcher.emit('change', 'change', null));
+  }
```

Comment:
```suggestion
```
cant we us `node:test` `mock.method(FSWatcher, 'change')` or something else that won't require adding a specific method used only by tests?

Reply:
You're right. I changed the concept.